fdbserver/storageserver.actor.cpp

```diff
@@ -997,7 +997,7 @@ struct StorageServer : public IStorageMetricsService {
 	std::unordered_map<UID, std::shared_ptr<MoveInShard>> moveInShards;
 
 	bool shardAware; // True if the storage server is aware of the physical shards.
-	Future<Void> auditSSShardInfoActor;
+	std::unordered_map<AuditType, std::pair<UID, ActorCollection>> ssAuditManager;
```

Comment:
nit: `*Manager` sounds more like a `class` responsible for `managing` some resources/tasks, this is more of a container, maybe `audits`?

Reply:
Agree. Thanks for suggestion!